src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -3,73 +3,108 @@
+ *     Finally, bins that contain only Ns are filtered out.
+ * </p>
  *
- * <p>Using the -P flag, the user can specify the amount of padding (in bp) added to each side of the intervals.
- * This padding is in addition to the padding added by the -ip (or --interval-padding) argument of
- * IntervalArgumentCollection. However, we encourage using only the -P flag.
+ * <h3>Input</h3>
  *
- * <p>The user can also specify the length of the bins (in bp) using the -BL option. If this is not commensurate with
- * the length of the padded intervals, then the last bin will be of different length than the others.  If zero is
- * specified, then no binning will be performed.</p>
+ * <ul>
+ *     <li>
+ *         Reference file.
+ *     </li>
+ *     <li>
+ *         Intervals to be preprocessed.
+ *         The argument {@code interval-merging-rule} must be set to {@link IntervalMergingRule#OVERLAPPING_ONLY}
+ *         and all other common arguments for interval padding or merging must be set to their defaults.
+ *         If no intervals are specified, then each contig will be assumed to be a single interval and binned accordingly;
+ *         this produces bins appropriate for whole genome sequencing analyses.
+ *     </li>
+ *     <li>
+ *         Padding length (in bp).
+ *         Use {@code padding} to specify the size of each of the regions added to both ends of the intervals that result
+ *         after overlapping intervals have been merged.  Do not use the common {@code interval-padding} argument, which must
+ *         instead be set to zero.  Intervals that would overlap after padding by the specified amount are instead only
+ *         padded until they are adjacent.
+ *     </li>
+ *     <li>
+ *         Bin length (in bp).
+ *         If this length is not commensurate with the length of a padded interval, then the last bin will be of
+ *         different length than the others in that interval.  If zero is specified, then no binning will be performed;
+ *         this is generally appropriate for targeted analyses.
+ *     </li>
+ * </ul>
  *
- * <p> The -O argument specifies a filename for the output bins, stored as a Picard interval list. </p>
+ * <h3>Output</h3>
+ *
+ * <ul>
+ *     <li>
+ *         Preprocessed Picard interval-list file.
+ *     </li>
+ * </ul>
+ *
+ * <h3>Examples</h3>
+ *
+ * <pre>
+ *     gatk PreprocessIntervals \
+ *          -R reference.fa \
+ *          -L intervals.interval_list \
+ *          --bin-length 0 \
+ *          --padding 250 \
+ *          -O preprocessed_intervals.interval_list
+ * </pre>
  *
- * <h3>Example</h3>
  * <pre>
- * gatk-launch --javaOptions "-Xmx4g" PreprocessIntervals \
- *   -R ref_fasta.fa \
- *   -L intervals.list \
- *   -BL 10000 \
- *   -P 500 \
- *   -O preprocessed-intervals.interval_list
+ *     gatk PreprocessIntervals \
+ *          -R reference.fa \
+ *          --bin-length 1000 \
+ *          --padding 0 \
```

Comment:
Does it make a difference to turn off padding in this instance? I should think the bins would take precedence. If so, then let's remove the `--padding 0 \`.

Reply:
You could imagine preprocessing WES targets that you'd want to both pad and break up into smaller bins (if your targets were very long, this might buy you a bit of resolution), which is why we allow for both arguments to be specified simultaneously.